src/BrazeEndOfArticleComponent.ts

```diff
@@ -7,12 +7,13 @@ import {
 } from './NewsletterEpic';
 
 import { COMPONENT_NAME as US_NEWSLETTER_EPIC_NAME, USNewsletterEpic } from './USNewsletterEpic';
-
 import { COMPONENT_NAME as AU_NEWSLETTER_EPIC_NAME, AUNewsletterEpic } from './AUNewsletterEpic';
-
 import { COMPONENT_NAME as UK_NEWSLETTER_EPIC_NAME, UKNewsletterEpic } from './UKNewsletterEpic';
-
 import { COMPONENT_NAME as EPIC_NAME, Epic } from './Epic';
+import {
+    COMPONENT_NAME as EPIC_WITH_HEADER_IMAGE_NAME,
+    EpicWithHeaderImage,
+} from './EpicWithHeaderImage';
```

Comment:
Super minor point, but this name kind of doesn't apply anymore! Now it's an epic with a bespoke title header. It does have an image in so this isn't totally misleading! Fine for this test, maybe let's double check our thoughts if we go any further with templating this approach?

Reply:
Ha yeah that's a good point, I think I'd managed to convince myself it was okay because like you say there is still an image in there. I'll give some thought to what a good name would be. I'm hoping this template will be short lived so sticking with the current name doesn't seem _that_ bad, would be nice to do better though.

Reply:
I've gone with `EpicWithSpecialHeader` for the name, which I think is a slight improvement on `EpicWithHeaderImage`.